packages/beacon-node/src/metrics/server/http.ts

```diff
@@ -72,12 +73,17 @@ export async function getHttpMetricsServer(
 
   const activeSockets = new HttpActiveSocketsTracker(server, socketsMetrics);
 
-  const {port, address} = opts;
-  logger.info("Starting metrics HTTP server", {port, address: address ?? "127.0.0.1"});
-
   await new Promise<void>((resolve, reject) => {
-    server.once("error", reject);
-    server.listen(port, address, resolve);
+    server.once("error", (err) => {
+      logger.error("Error starting metrics HTTP server", opts, err);
+      reject(err);
+    });
+    server.listen(opts.port, opts.address, () => {
+      const {port, address: host, family} = server.address() as AddressInfo;
+      const address = `http://${family === "IPv6" ? `[${host}]` : host}:${port}`;
```

Comment:
IPv6 is supported, this just ensures we print out a valid URL. 

Same approach is used by fastify
- https://github.com/fastify/fastify/pull/4902